heroku/cli
# refactor(pg-v5): Move command `pg:credentials:destroy` to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001b2PsbYAE/view

This converts `pg:credentials:destroy` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:credentials:destroy DB_NAME -a APP -n CRED ` with an app that has a pg db named DB_NAME and a credential named CRED that you can safely delete (to create one, just use `pg:credentials:create`)
3) Verify the command succeeds, deleting the credential
4) Compare with prod
